### PR TITLE
Try older version of COS image for ci-containerd-e2e-ubuntu-gce test

### PR DESCRIPTION
**Pull request by dims**
Testing a theory from @endocrimes and @bobbypage from:
https://github.com/kubernetes/kubernetes/issues/111517#issuecomment-1198584691

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

**Comment by bobbypage** [comment]
I think this will change it for all the jobs in `config/jobs/kubernetes/sig-node/containerd.yaml`, can we just change it for the `ci-containerd-e2e-ubuntu-gce` job?

**Comment by dims** [comment]
@bobbypage sure

**Comment by bobbypage** [comment]
To do that, I think all we need is 

`--env=KUBE_GCE_MASTER_IMAGE =cos-85-13310-1308-1`
`--env=KUBE_GCE_MASTER_PROJECT=cos-cloud`

in the job config

File: config/jobs/kubernetes/sig-node/containerd.yaml [hunk 1]
```diff
@@ -134,6 +134,7 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --check-leaked-resources
+      - --env=KUBE_GCI_VERSION=cos-85-13310-1308-1
```
Comment (bobbypage) [inline config/jobs/kubernetes/sig-node/containerd.yaml:137]: hmm, will the also affect the node image? I think we still want the node image to be ubuntu, but have the master image be older version of COS?

I'm not sure behavior of passing in `KUBE_GCI_VERSION` and `--gcp-node-image=ubuntu` via kubetest...

Comment (bobbypage) [reply config/jobs/kubernetes/sig-node/containerd.yaml:137]: If we just want to change the master image back to older COS, I think we can just change `KUBE_GCE_MASTER_IMAGE` and `KUBE_GCE_MASTER_PROJECT`

Comment (dims) [reply config/jobs/kubernetes/sig-node/containerd.yaml]: Done!

**Comment by bobbypage** [comment]
Thanks! Let's see if it helps! 

/lgtm

**Comment by dims** [comment]
@bobbypage done! peeked at https://cs.k8s.io/?q=KUBE_GCE_MASTER_IMAGE&i=nope&files=&excludeFiles=&repos=kubernetes/kubernetes and it seems right. 

thanks!

**Pull request merged**